homework5/task01/homework.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+
+class TimeToSolveError(ValueError):
+    """Days to solve task can't be negative"""
+
+
+class Homework:
+    """Class for storing information about task and its time to complete"""
+    def __init__(self, text: str, days_to_solve: int) -> None:
+        if days_to_solve < 0:
+            raise TimeToSolveError
+
+        self.text = text
+        self.created = datetime.datetime.now()
+        self.final_day = self.created + datetime.timedelta(days=days_to_solve)
+
+    @property
+    def deadline(self) -> datetime.timedelta:
```

Comment:
deadline rather means how many days are set from the date homework was created.. but OK